cdist/conf/type/__letsencrypt_cert/gencode-remote

```diff
@@ -0,0 +1,18 @@
+domain="$__object_id"
+
+exists=$(cat "$__object/explorer/exists")
+webroot="$(cat "$__object/parameter/webroot")"
+admin_email="$(cat "$__object/parameter/admin-email")"
+
+if [ -n "$exists" ]; then
+	exit 0
+fi
+
+cat <<EOF
+if [ ! -d "$webroot" ]; then
+	echo mkdir -p "$webroot"
```

Comment:
@AnotherKamila Shouldn't the above be without echo? It's inside here string.

Reply:
Oh. Um. Yes.